src/main/java/Task.java

```diff
@@ -0,0 +1,56 @@
+public class Task {
+    private String description;
+    private boolean isCompleted;
+    private int index;
+    public static int currentIndex = 0;
```

Comment:
Would it be possible to shift this variable to the main code instead? Since that is where the array containing the Tasks is. Unless there is a specific reason for having the currentIndex here